pages/newuser.tsx

```diff
@@ -34,18 +43,57 @@ export default function NewUserPage({
         <title>Wannado</title>
         <link rel="shortcut icon" href="/favicon.ico" />
       </Head>
-      <Header />
+      <Header heading={"Add New Artist"} />
+
       <StyledForm onSubmit={handleSubmit}>
-        <input name="artistname" placeholder="Artistname" required />
-        <input name="city" placeholder="city" required />
-        <input name="streetname" placeholder="streetname" required />
-        <input name="number" placeholder="number" required />
-        <input name="pics" type="file" multiple required />
-        <Button
-          name={`${isloading ? "Uploading" : "Sumbit"}`}
-          inactive={isloading ? true : false}
-        ></Button>
+        <StyledLabel>
+          <span>Artistname:</span>
+          <input name="artistname" required />
+        </StyledLabel>
+        <StyledLabel>
+          <span>City:</span>
+          <input name="city" required />
+        </StyledLabel>
+        <StyledLabel>
+          <span>Street:</span>
+          <input name="streetname" required />
+          <input name="number" placeholder="#" type="number" required />
+        </StyledLabel>
+
+        {selectedImages.length === 4 ? (
+          <Button
+            name={`${isloading ? "Uploading" : "Sumbit"}`}
+            inactive={isloading ? true : false}
+          ></Button>
```

Comment:
ist mehr QA, aber ich hab mir die Preview angeschaut und ich kann diesen Button nicht sehen

Reply:
und wenn man auf Image Upload klickt und dann auf abbrechen, dann kommt eine Fehlermeldung...
`TypeError: Failed to execute 'createObjectURL' on 'URL': Overload resolution failed.
    at newuser-164688a0657ad768.js:1:9344
    at Array.map (<anonymous>)
    at C (newuser-164688a0657ad768.js:1:9296)
    at ak (framework-3b5a00d5d7e8d93b.js:9:60917)
    at up (framework-3b5a00d5d7e8d93b.js:9:72803)
    at i (framework-3b5a00d5d7e8d93b.js:9:120372)
    at oD (framework-3b5a00d5d7e8d93b.js:9:99114)
    at framework-3b5a00d5d7e8d93b.js:9:98981
    at oO (framework-3b5a00d5d7e8d93b.js:9:98988)
    at oE (framework-3b5a00d5d7e8d93b.js:9:95740)`

Reply:
Der Button ist erst sichtbar wenn vier img-files zum upload ausgewählt wurden. 

Ja und der Bug ist ärgerlich. Da muss ich mal reinschauen

Reply:
Okay fixed it. War ein fehlender Check innerhalb der changeImage-Funktion:

```typescript  
function changeImage(event: React.SyntheticEvent) {
    const input = (event.target as HTMLInputElement).files;
    if (input && input.length > 0) {
      setSelectedImages([...selectedImages, input![0]]);
    }
  }
```